server/channels/app/bot_test.go

```diff
@@ -112,6 +112,57 @@ func TestCreateBot(t *testing.T) {
 	})
 }
 
+func TestEnsureBot(t *testing.T) {
+	t.Run("ensure bot should pass if already exist bot user", func(t *testing.T) {
+		th := Setup(t).InitBasic()
+		defer th.TearDown()
+		pluginId := "pluginId"
+
+		PluginErr := th.App.SetPluginKey(pluginId, "key", []byte("test"))
+		assert.Nil(t, PluginErr)
+
+		_, err := th.App.EnsureBot(th.Context, pluginId, &model.Bot{
+			Username:    "username",
+			Description: "a bot",
+			OwnerId:     th.BasicUser.Id,
+		})
+		require.Nil(t, err)
+
+		_, err = th.App.EnsureBot(th.Context, pluginId, &model.Bot{
+			Username:    "username",
+			Description: "a bot",
+			OwnerId:     th.BasicUser.Id,
+		})
+		require.Nil(t, err)
+	})
+
```

Comment:
It is not strictly related to your PR, but it would be great to have a test that ensures that the bot gets updated if the description changes.